jetty-util/src/main/java/org/eclipse/jetty/util/component/AbstractLifeCycle.java

```diff
@@ -234,23 +234,4 @@ public void setStopTimeout(long stopTimeout)
         @Override public void lifeCycleStopped(LifeCycle event) {}
         @Override public void lifeCycleStopping(LifeCycle event) {}
     }
-    
```

Comment:
This removal seems unrelated to the issue.

Reply:
It's related because I would have needed to add `if (e != cause)`, but then figured out that it's an unused field, so I removed it.